clang/test/Preprocessor/predefined-macros-hlsl.hlsl

```diff
@@ -12,7 +12,7 @@
 // HALF: #define __HLSL_ENABLE_16_BIT 1
 // NOHALF-NOT: __HLSL_ENABLE_16_BIT
 
-// CHECK: #define __HLSL_VERSION 2021
+// CHECK: #define __HLSL_VERSION 2028
```

Comment:
2028 == 202x and 2029 == 202y I expect?

Reply:
Yea. Subject to revision once we have an actual year.